.github/ISSUE_TEMPLATE/2.feature_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Feature Request
+description: I have a suggestion
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: Please use a concise and distinct title for the issue.
+  - type: textarea
+    attributes:
+      label: Pitch
+      description: Describe your idea for a feature. Make sure it has not already been suggested/implemented/turned down before.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Motivation
+      description: Why do you think this feature is needed? Who would benefit from it?
+    validations:
+      required: true
```

Comment:
Does this need to be two fields?

Reply:
I think it will help fend off some requests without any thought behind it. The second field will make people think a bit longer before submitting.